.github/workflows/ci_release.yml

```diff
@@ -83,38 +86,47 @@ jobs:
   tests:
     uses: ./.github/workflows/tests.yml
     with:
-      ANSYS_VERSION: "241"
+      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '242' }}
       python_versions: '["3.9", "3.10", "3.11"]'
       wheel: true
       wheelhouse: true
-      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.sp01' }}
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
     secrets: inherit
 
   tests_any:
     uses: ./.github/workflows/tests.yml
     with:
-      ANSYS_VERSION: "241"
+      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '242' }}
       python_versions: '["3.9", "3.10", "3.11"]'
       wheel: true
       wheelhouse: false
-      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.sp01' }}
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
       test_any: true
     secrets: inherit
 
   docs:
     uses: ./.github/workflows/docs.yml
     with:
-      ANSYS_VERSION: "241"
-      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.sp01' }}
+      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '242' }}
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
       event_name: ${{ github.event_name }}
     secrets: inherit
 
   examples:
     uses: ./.github/workflows/examples.yml
     with:
-      ANSYS_VERSION: "241"
+      ANSYS_VERSION: ${{ github.event.inputs.ansys_version || '242' }}
       python_versions: '["3.9", "3.10", "3.11"]'
-      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.sp01' }}
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
+    secrets: inherit
+
+  retro_241:
+    name: "retro 241"
+    uses: ./.github/workflows/tests.yml
+    with:
+      ANSYS_VERSION: "241"
+      python_versions: '["3.9"]'
+      DOCSTRING: false
```

Comment:
@PProfizi retro 241 should point to sp01, right? (unless 241 now is equal to sp01)

Reply:
@rafacanton good catch!